File: MDC/mdcWEB/mdc.js

function calcularMDC(n1, n2){
    while (n2 !== 0) {
        let aux = n2;
        n2 = n1 % n2;
        n1 = aux;
    }
    return n1;
}

let n1 = 16;
let n2 = 24;
document.write(`o mdc é ${calcularMDC(n1, n2)}`)

File: MDC/mdcJAVA/Mdc.java
package MDC.mdcJAVA;

public class Mdc {
    public int num1;
    public int num2;

    public int calculoDoMDC(){
        while (num2 != 0){
            int aux = this.num2;
            this.num2 = this.num1  % this.num2;
            this.num1 = aux;
        }
        return this.num1;
    }
}